interfaces/HO-Portal/src/app/program/submit-payment-popup/submit-payment-popup.component.html

```diff
@@ -0,0 +1,21 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title>
+      {{
+      'page.program.program-people-affected.actions.do-payment' | translate
+      }}
+      #{{ submitPaymentProps.payment }}
```

Comment:
See AB#12126.

Should be handled like: https://github.com/global-121/121-platform/pull/2696/files#diff-f86a2850991fe920bcd73b63eb7df703ba24f756d40f07ddf2ead04a11b823eaL109-L112